index.js

```diff
@@ -0,0 +1,380 @@
+	}
+});
+app.get("/dashboard_facility_card", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking;", (err, bookingData) => {
+			con.query("select * from events;", (err, eventsData) => {
+				res.render("dashboard_facility_card", {
+					res: req.session.user,
+					bookingData,
+					eventsData
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/dashboard_facility_table", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking;", (err, bookingData) => {
+			con.query("select * from events;", (err, eventsData) => {
+				res.render("dashboard_facility_table", {
+					res: req.session.user,
+					bookingData,
+					eventsData
+				});
+			});
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/request", (req, res) => {
+	if (req.session.user) {
+		res.render("request");
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/requestUpdate", (req, res) => {
+	if (req.session.user) {
+		con.query("select * from booking where id=4;", (err, bookingData) => {
+			con.query(
+				"select * from events where id=30;",
+				(err, eventsData) => {
+					res.render("requestUpdate", {
+						res: req.session.user,
+						bookingData,
+						eventsData
+					});
+				}
+			);
+		});
+	} else {
+		res.redirect("/login");
+	}
+});
+app.get("/logout", (req, res) => {
+	res.clearCookie("session", {
+		path: "/"
+	});
+	res.redirect("/login");
+});
+app.post("/onLogin", (req, res) => {
+	var username = req.body.username;
+	var password = req.body.password;
+	con.query(
+		"select * from users where username='" + username + "';",
+		(err, rows) => {
+			if (!err) {
+				if (rows.length > 0) {
+					if (
+						password ===
+						AES.decrypt(rows[0].password, `${process.env.AES_key}`)
+					) {
+						req.session.user = rows[0];
+						res.redirect("/dashboard");
+					} else {
+						console.log("Invalid password");
+						res.status(401).render("error", {
+							errorMessage: "Authentication error!"
+						});
+					}
+				} else {
+					console.log("Username not found!");
+					res.status(401).render("error", {
+						errorMessage: "Authentication error!"
+					});
+				}
+			} else {
+				console.log("Username not found!");
+				res.status(401).render("error", {
+					errorMessage: "Authentication error!"
+				});
+			}
+		}
+	);
+});
+app.post("/checkAvailability", (req, res) => {
+	if (req.session.user) {
+		con.query(
+			"SELECT slotHall FROM booking WHERE event_id IN (SELECT id FROM events WHERE date_wanted = ?)",
+			req.body.date_wanted,
+			(err, result) => {
+				if (!err) {
+					res.send(result);
+				} else {
+					console.log(err);
+					res.status(500).render("error", {
+						errorMessage: "Availability checking failed!"
+					});
+				}
+			}
+		);
+	} else {
+		res.redirect("/login");
+	}
+});
+app.post("/makeRequest", (req, res) => {
+	if (req.session.user) {
+		var userID = req.session.user.id;
+		var date_wanted = req.body.date_wanted;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'date_wanted' is not in camel case. (camelcase)](https://app.codacy.com/app/AravindVNair99/Hall-Management-System/pullRequest?prid=4592231)